.github/workflows/main.yml

```diff
@@ -169,9 +170,16 @@ jobs:
           mkdir -p OBS.app/Contents/MacOS
           mkdir OBS.app/Contents/PlugIns
           mkdir OBS.app/Contents/Resources
+          mkdir OBS.app/Contents/Frameworks
 
           cp rundir/RelWithDebInfo/bin/obs ./OBS.app/Contents/MacOS
           cp rundir/RelWithDebInfo/bin/obs-ffmpeg-mux ./OBS.app/Contents/MacOS
+          if ! [ "${{ env.CEF_BUILD_VERSION }}" -le 3770 ]; then
+            cp -R "rundir/RelWithDebInfo/bin/OBS Helper.app" "./OBS.app/Contents/Frameworks/OBS Helper.app"
+            cp -R "rundir/RelWithDebInfo/bin/OBS Helper (GPU).app" "./OBS.app/Contents/Frameworks/OBS Helper (GPU).app"
+            cp -R "rundir/RelWithDebInfo/bin/OBS Helper (Plugin).app" "./OBS.app/Contents/Frameworks/OBS Helper (Plugin).app"
+            cp -R "rundir/RelWithDebInfo/bin/OBS Helper (Renderer).app" "./OBS.app/Contents/Frameworks/OBS Helper (Renderer).app"
```

Comment:
After some thinking, I believe it'd be best to title these "OBS Browser Helper" as they're specific to the browser, and will provide better insight for user's trying to troubleshoot issues.

Reply:
CEF expects the helpers to be called `APPNAME Helper (Function).app`. It's the law.

Reply:
Here's the official word on it - while there is a way to rename the helper (as we do for `obs-browser-page`) this was discouraged for use on macOS: https://groups.google.com/g/cef-announce/c/YK6CTvGNhlY/discussion